addons/stock/models/stock_move.py

```diff
@@ -133,7 +133,7 @@ def _default_group_id(self):
         'Propagate cancel and split', default=True,
         help='If checked, when this move is cancelled, cancel the linked move too')
     delay_alert_date = fields.Datetime('Delay Alert Date', help='Process at this date to be on time', compute="_compute_delay_alert_date", store=True)
-    picking_type_id = fields.Many2one('stock.picking.type', 'Operation Type', check_company=True)
+    picking_type_id = fields.Many2one('stock.picking.type', 'Operation Type', compute="_compute_picking_type_id", store=True, check_company=True)
```

Comment:
check_company seems useless now regarding the way the field is computed

Reply:
Well, the picking_type_id is a compute store and we can imagine to set it with something else (where picking_id or production_id is False). By example in the test we set picking_type_id manually. I prefer to keep it by security and the cost is near of zero.